install/kubernetes/cilium/templates/cilium-agent-daemonset.yaml

```diff
@@ -458,6 +479,16 @@ spec:
           type: DirectoryOrCreate
         name: bpf-maps
 {{- end }}
+      # To mount cgroup filesystem on the host
+      - hostPath:
+          path: /proc
+          type: DirectoryOrCreate
+        name: host-proc
```

Comment:
We shouldn't mount host's `/proc` into Cilium's DaemonSet.

Reply:
A bit similar as above, this is missing _why_.